integration-tests/test/fee-payment.spec.ts

```diff
@@ -29,58 +28,6 @@ describe('Fee Payment Integration Tests', async () => {
     )
   })
 
-  it(`Should return a gasPrice of ${TxGasPrice.toString()} wei`, async () => {
-    const gasPrice = await env.l2Wallet.getGasPrice()
-    expect(gasPrice).to.deep.eq(TxGasPrice)
-  })
-
-  it('Should estimateGas with recoverable L2 gasLimit', async () => {
```

Comment:
I don't think its in scope to remove this functionality with this PR

Reply:
Out of scope to remove the functionality or the integration test? It's basically required to remove the functionality or end up having to add a bunch of ugly geth code to parse the L2 gas value out. If we're gonna get rid of this gas mechanism anyway I'd rather toss it here and save myself the work.

Reply:
So what you are saying is that you would like to remove the L1 portion of the fee for now?